internal/pkg/flags/flags.go

```diff
@@ -52,6 +52,8 @@ const (
 	DefaultWatcherResourceLimitsMemory                                  = "200Mi"
 	DefaultDropStoredVersion                                            = "v1alpha1"
 	DefaultMetricsCleanupIntervalInMinutes                              = 15
+	DefaultKlmControllerManagerName                                     = "klm-controller-manager"
```

Comment:
we don't need to define owner as flag, owner can be a static variable, not necessary to be configured, please check [here](https://github.com/kyma-project/lifecycle-manager/blob/9be2538f0201fe61eb7ac1f620242137f6178c74/internal/controller/watcher_controller.go#L191) how we setup owner for other resources, use `shared.OperatorName`

Reply:
to be discussed with the team tomorrow 👍🏻

Reply:
Agreed that we are going to use `WatcherCR` as the owner instead. This will make a big part of this PR obsolete. Will consider to close this one in favor of another more streamlined one.